dashboard/lib/services/lti.rb

```diff
@@ -238,6 +238,9 @@ def self.sync_section_roster(lti_integration, lti_section, nrps_section)
         end
       end
 
+      # Unarchive archived sections, even if there are no changes
+      section.update(hidden: false)
```

Comment:
Minor optimization to save us from doing the update if the section isn't archived:
```suggestion
      section.update(hidden: false) if section.hidden
```